AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -382,7 +387,7 @@ def test_FBXBlackboxTest_AssetInfoModified_AssetReprocessed_ResultInExpectedProd
         self.run_fbx_test(workspace, ap_setup_fixture,
                           asset_processor, project, blackbox_param, True)
 
-
+    @pytest.mark.SUITE_sandbox
```

Comment:
Does this need to be added for this function?  It looks like its just a helper function.  Same for the below function

Reply:
Removed in the following commit.